webapp/tsconfig.json

```diff
@@ -25,6 +27,9 @@
     "experimentalDecorators": true,
     "jsx": "react"
   },
+  "files": [
+    "src/custom.d.ts"
+  ],
```

Comment:
I don't think I've ever used `files` like this for `.d.ts` files. Maybe we can move `custom.d.ts` into `types/svg.d.ts`, and then set tsconfig's `typeRoots` to `./types`. I think that's the proper way to do this.

But if it ain't broke, don't fix it? 🤷

Reply:
Let's do it the proper way.